src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -1,116 +1,142 @@
 <div class="course-info-bar" *ngIf="exercise" id="exercise-header">
     <div class="row">
-        <div class="col general-info">
+        <div class="col-auto general-info">
             <h3>
                 <fa-icon *ngIf="displayBackButton" [icon]="faArrowLeft" (click)="onBackClick()" class="back-button me-2"></fa-icon>
+                <fa-icon *ngIf="exercise.type" [icon]="getIcon(exercise.type)" [ngbTooltip]="getIconTooltip(exercise.type) | artemisTranslate" container="body"></fa-icon>
                 <ng-content select="[pagetitle]"></ng-content>
                 <jhi-not-released-tag [exercise]="exercise"></jhi-not-released-tag>
             </h3>
-
+            <h4 class="row" *ngIf="exercise.maxPoints">
+                <span>
+                    {{ 'artemisApp.courseOverview.exerciseDetails.points' | artemisTranslate }}
+                    {{ exercise.maxPoints }}
+                    <span *ngIf="exercise.bonusPoints">({{ 'artemisApp.courseOverview.exerciseDetails.bonus' | artemisTranslate }} {{ exercise.bonusPoints }})</span>
+                    <jhi-included-in-score-badge [includedInOverallScore]="exercise.includedInOverallScore"></jhi-included-in-score-badge>
+                </span>
+            </h4>
+            <h4 class="row" *ngIf="exercise.assessmentType">
+                <span>
+                    {{ 'artemisApp.courseOverview.exerciseDetails.assessmentType' | artemisTranslate }}
+                    {{ exercise.assessmentType }}
+                </span>
+            </h4>
+            <h4 class="d-md-flex" *ngIf="exercise.categories || exercise.difficulty">
+                <span *ngFor="let category of exerciseCategories">
+                    <span class="badge text-white me-1" [ngStyle]="{ backgroundColor: category.color }">{{ category.category }}</span>
+                </span>
+                <span *ngIf="exercise.difficulty">
+                    <jhi-difficulty-badge class="me-1" [exercise]="exercise" [showNoLevel]="false"></jhi-difficulty-badge>
+                </span>
+            </h4>
+            <h4 class="row" *ngIf="submissionPolicy">
+                <span>
+                    {{ 'artemisApp.programmingExercise.submissionPolicy.submissionLimitTitle' | artemisTranslate }}:
+                    {{
+                        submissionPolicy.submissionLimit +
+                            (submissionPolicy.exceedingPenalty
+                                ? ' (' +
+                                  ('artemisApp.programmingExercise.submissionPolicy.submissionPenalty.penaltyInfoLabel' | artemisTranslate) +
+                                  ': ' +
+                                  submissionPolicy.exceedingPenalty +
+                                  'P' +
+                                  ')'
+                                : '')
+                    }}
```

Comment:
I am pretty sure this code could be greatly improved by passing submissionPolicy.exceedingPenalty to artemisTranslate when translating `artemisApp.programmingExercise.submissionPolicy.submissionPenalty.penaltyInfoLabel`.

Reply:
Done